state-chain/runtime/src/runtime_apis.rs

```diff
@@ -66,6 +69,14 @@ pub struct BoostPoolDepth {
 	pub available_amount: AssetAmount,
 }
 
+#[derive(Encode, Decode, Eq, PartialEq, TypeInfo, Serialize, Deserialize)]
+pub struct BoostPoolDetails {
+	pub available_amount: AssetAmount,
+	pub amounts: BTreeMap<AccountId32, AssetAmount>,
```

Comment:
I think we can simply return one property `available_amounts: BTreeMap<AccountId32, AssetAmount>` instead of 2 separate.